### PR TITLE
Introduce CONTRIBUTING.md

### PR DESCRIPTION
**Pull request by hanefi**
I used a copy of Citus' CONTRIBUTING.md as a starting point.

I removed sections that are not relevant to pgcopydb, and added some more sections about documentation and running tests.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,84 @@
...
+We're happy you want to contribute! You can help us in different ways:
+
+* Open an [issue](https://github.com/dimitri/pgcopydb/issues) with suggestions
+  for improvements, potential bugs, etc.
+* Fork this repository and submit a pull request
+
+### Building from source code
+
+Follow the relevant docs at our
+[README.md](https://github.com/dimitri/pgcopydb/#installing-pgcopydb)
```
Comment (dimitri) [inline CONTRIBUTING.md:12]: I think the README should be as short as possible, and now that we have build/install coverage in the main documentation at https://pgcopydb.readthedocs.io/en/latest/install.html#build-from-sources I'm wondering if we should remove the section from the README, and in this PR point to the docs. What do you think?

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+This tool uses `uncrustify` under the hood.
+
+
+```bash
+# Use your favorite package manager, and install `uncrustify` by running one of
+# the following
+apt-get install uncrustify
+brew install uncrustify
+apk add --upgrade uncrustify
+dnf install uncrustify # may require epel-release rpm
```
Comment (dimitri) [inline CONTRIBUTING.md:29]: Let's avoid showing the alternatives within a single bash script, (some) people will inevitably copy/paste the whole thing and wonder about it. Might even report an issue...

**Review by dimitri — CHANGES_REQUESTED** [review]
Thanks a lot Hanefi, we need such a file!

File: CONTRIBUTING.md (hunk 1 above)
Comment (hanefi) [reply CONTRIBUTING.md]: I read through that piece of the README, and realized that yum installation is outdated for a while. I think it is a good idea to remove installation instructions completely, and add a reference to our docs.

I also took the liberty of adding a reference in our docs for building from source to CONTRIBUTING.md.

**Review by dimitri — APPROVED** [review]

**Pull request merged**